cli/reporter/reporter.go

```diff
@@ -76,6 +78,7 @@ func New(typ string) (*Reporter, error) {
 }
 
 func (r *Reporter) Print(data *explorer.ReportCollection, out io.Writer) error {
+	logger.DebugDumpYAML("report_collection", data)
```

Comment:
Left-over from debugging?

Reply:
This is intentional. It allows us to debug the reporting. It only writes down the reports when DEBUG=1 is used.